src/foundry/foundry.js/pixi/containers/placeableObjects/token.d.ts

```diff
@@ -577,68 +493,52 @@ declare global {
     protected _onDragLeftMove(event: PIXI.InteractionEvent): void;
 
     /**
-     * A factory method to create a Token instance from an Actor entity.
-     * The Token is not automatically saved to the database, it is up to the caller whether or not they wish to do that.
-     *
-     * @param actor     - The input actor entity
-     * @param tokenData - Additional data, such as x, y, rotation, etc. for the created token
-     * @returns The created Token instance
+     * @deprecated since 0.8.0
      */
-    static fromActor(actor: Actor, tokenData?: DeepPartial<Token.Data>): Promise<Token>;
+    static fromActor(
+      actor: InstanceType<ConfiguredDocumentClass<typeof Actor>>,
+      tokenData?: InstanceType<ConfiguredDocumentClass<typeof TokenDocument>>['data']['_source']
+    ): never;
 
     /**
-     * @deprecated since 0.7.4
+     * A helper method to retrieve the underlying data behind one of the Token's attribute bars
+     * @param barName     - The named bar to retrieve the attribute for
+     * @param alternative - An alternative attribute path to get instead of the default one
+     * @returns The attribute displayed on the Token bar, if any
+     * @deprecated since 0.8.0
      */
-    shiftPosition(dx: number, dy: number): Promise<this>;
+    getBarAttribute(
+      barName: string,
+      { alternative }?: { alternative?: string }
+    ): {
+      type: 'bar' | 'value';
+      attribute: string;
+      value: number;
+      max?: number;
+    } | null;
+
+    /** @override */
+    clone(): this;
+
+    /** @override */
+    get bounds(): NormalizedRectangle;
```

Comment:
where does this come from? I don't see it in foundry

Reply:
Removed the `abstract` from the `PlaceableObject.bounds()`